static/css/base.css

```diff
@@ -1,165 +1,175 @@
 @font-face {
-  font-family: "DINOffcCondMedi";
-  src: url('../fonts/DINOffc-CondMedi.ttf');
+	font-family: "DINOffcCondMedi";
```

Comment:
It looks like you changed the indent formatting to use a tab character instead of two spaces. I'm not sure if this was intentional or if it was something that your editor automatically did, but in the future, if you make a formatting change to the code, please put formatting changes into their own commit and write in the commit message that it was a formatting change.

The reason for this is that it is confusing when reviewing a commit which has both formatting changes and content changes, as the formatting changes look similar to content changes in the diff. 

Also, I'm going to ask Marie if we can create a code style repository for CNDLS styles so we can all follow a standard for indentation, spacing, etc for each language. I'll keep you updated on that.